plugins/modules/htpasswd.py

```diff
@@ -233,20 +208,12 @@ def main():
     create = module.params['create']
     check_mode = module.check_mode
 
-    if not passlib_installed:
-        module.fail_json(msg=missing_required_lib("passlib"), exception=PASSLIB_IMP_ERR)
+    deps.validate(module)
 
     # Check file for blank lines in effort to avoid "need more than 1 value to unpack" error.
```

Comment:
I wonder if this 'hack' below is still needed. Maybe this was necessary for an older version of passlib.

Reply:
Yup, maybe, but I will leave that for another PR, with tests supporting the hypothesis.

Reply:
Yes, definitely!